src/js/lib/jquery.kazoosdk.js

```diff
@@ -576,6 +576,14 @@
 				'deleteNotification': { verb: 'DELETE', url: 'accounts/{accountId}/notifications/{notificationId}' },
 				'getDnsEntries': { verb: 'GET', url: 'accounts/{accountId}/whitelabel/domains' },
 				'checkDnsEntries': { verb: 'POST', url: 'accounts/{accountId}/whitelabel/domains?domain={domain}&use_tcp={useTcp}' }
+			},
+			matchList: {
+				'list': { 'verb': 'GET', 'url': 'accounts/{accountId}/match_lists/' },
+				'create': { 'verb': 'PUT', 'url': 'accounts/{accountId}/match_lists' },
+				'get': { 'verb': 'GET', 'url': 'accounts/{accountId}/match_lists/{matchListId}' },
+				'update': { 'verb': 'POST', 'url': 'accounts/{accountId}/match_lists/{matchListId}' },
+				'patch': { 'verb': 'PATCH', 'url': 'accounts/{accountId}/match_lists/{matchListId}' },
+				'remove': { 'verb': 'DELETE', 'url': 'accounts/{accountId}/match_lists/{matchListId}' }
```

Comment:
Use `delete` as the key, instead of remove, as it is the standard name used for this action.

Reply:
Nice catch 😊 I'll fix that